### PR TITLE
Add CO Governor

### LINKED ISSUE #80: Create form for CO Governor: 7
https://www.colorado.gov/governor/share-your-comments 

Note the confidentiality checkbox.

### PR DESCRIPTION
**Pull request by jason44406**
Fixes #80

### CODE REVIEW

File: states/CO/governor.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+contact_form:
+  steps:
+    # Note that the form is inside of an iframe on this website: https://www.colorado.gov/governor/share-your-comments
+    - visit: "https://www.tfaforms.com/4679739?faIframeUniqueId=49cteb2vyx&hostURL=https%3A%2F%2Fwww.colorado.gov%2Fgovernor%2Fshare-your-comments"
```
Comment (jasonkb) [inline states/CO/governor.yaml]: Whoa this is... Salesforce Forms?! https://www.formassembly.com/

File: states/CO/governor.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    # This is a radio button asking if the comment is related to proposed legislation, and checks the "no" radio button.
+    - javascript:
+        - value: "document.querySelector('#tfa_2746').checked = true;"
```
Comment (jasonkb) [inline states/CO/governor.yaml]: Hm I think you can use `choose` directive here? From the contact-congress schema:

https://github.com/unitedstates/contact-congress/blob/master/documentation/schema.md#checkuncheckchoose

File: states/CO/governor.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          value: $EMAIL
+          required: true
+    - select:
+        - name: "tfa_71"
+          selector: "#tfa_71"
+          value: $ADDRESS_COUNTY
+          required: true
+    # This is the confidentiality statement that prevents correspondence from being publicly disclosed.  At the time of this design the box is checked by default, but this JS ensures that the checkbox is selected.
+    - javascript:
+        - value: "document.getElementById('tfa_2727').checked = true;"
```
Comment (jasonkb) [inline states/CO/governor.yaml]: Can this be a `check` directive?

**Comment by jasonkb** [comment]
Cool, Colorado!

**Comment by jason44406** [comment]
I had some trouble getting ```choose``` and ```check``` to hit the right element.  I'll review.

**Comment by jason44406** [comment]
Our 'choose' directive seems to be a random choice, which is inappropriate for this application.  I updated the config with the check directive instead of the JS.  It ran successfully in debugger.  Let me know if you have any other comments.

**Comment by jasonkb** [comment]
Ooof. Our choose directive is dumb : (

**Pull request merged**